code/controllers/subsystem/vote.dm

```diff
@@ -209,7 +209,7 @@ SUBSYSTEM_DEF(vote)
 
 /datum/controller/subsystem/vote/proc/initiate_vote(vote_type, initiator_key, code_invoked = FALSE)
```

Comment:
I ain't using the `initiator_key` since then somebody called `the server` can force a vote whenever if they manage to call this